index.html

```diff
@@ -6,42 +6,49 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
+   
     <title>Jonathan's Portfolio</title>
 </head>
-
-<body>
-
-
     <body>
-        <div class="intro-card">
+       
             <header>
                 <div class="nav-container">
                     <div class="nav-logo">
                         <a href="#">My Logo</a>
                     </div>
                     <div class="desktop-tools">
-                        <ul class="flex">
+                        <nav class="content">
+                            <ul class="navnav-menu">
+                            <li class="navnav-item">
+                                <a href="#" class="link">Portfolio</a>
+                            </li>
+                            <li>
+                                <a href="#" class="link">About</a>
+                            </li>
                             <li>
-                                <a href="#">Portfolio</a>
+                                <a href="#" class="link">Contact</a>                            
+                            </li>
+                            </ul>
+                        </nav>
```

Comment:
- [ ] Let's kindly make sure that when the user clicks any of the mobile menu options, the correct part of the page is displayed as per the requirement.